api/audit/views.py

```diff
@@ -44,7 +44,9 @@ def get_queryset(self):
                 self.request.query_params.get("environment")
             )
             q = q & (Q(environment__id=environment_id) | Q(environment=None))
-        return AuditLog.objects.filter(q)
+        return AuditLog.objects.filter(q).select_related(
```

Comment:
I think it would be worth adding in a test that calls this endpoint and checks the number of queries that are made. 

You can either use `TransactionTestCase` or the `django_assert_num_queries` fixture to then do something like: 

```python
with self.assertNumQueries(2):  # I think we'll need 2: 1 to do the count for paging, one to get the records
    response = self.client.get(...)
```